topics/proteomics/tutorials/clinical-mp-data-interpretation/tutorial.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: tutorial_hands_on
+
+title: Clinical-MP-Database Interpretation
+zenodo_link: '(https://doi.org/10.5281/zenodo.10105821)'
+questions:
+- Why do we need to interpret the data?
+- Can we visualize the data?
+objectives:
+- Perform group comparison analysis.
+- Analyze significant proteins
+- Look at the taxonomic distribution of the quantified peptides
+time_estimation: 3H
+key_points:
+- Data interpretation and visualization of clinical samplles.
+contributors:
+  - subinamehta
+  - pratikdjagtap
+  - timothygriffin
+  - katherine-d21
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.